_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -3091,18 +3090,7 @@ private static function core_get_plugins() {
 	 * }
 	 */
 	public static function get_service_api_key( $request ) {
-		$service = self::validate_service_api_service( $request['service'] );
-		if ( ! $service ) {
-			return self::service_api_invalid_service_response();
-		}
-		$option  = self::key_for_api_service( $service );
-		$message = esc_html__( 'API key retrieved successfully.', 'jetpack' );
-		return array(
-			'code'            => 'success',
-			'service'         => $service,
-			'service_api_key' => Jetpack_Options::get_option( $option, '' ),
-			'message'         => $message,
-		);
+		return WPCOM_REST_API_V2_Endpoint_Service_API_Keys::get_service_api_key( $request );
 	}
```

Comment:
Do we even still need this method? Or can we drop it here, and just refer to `WPCOM_REST_API_V2_Endpoint_Service_API_Keys::get_service_api_key` (instead of ` __CLASS__ . '::get_service_api_key'`) on [R435](https://github.com/Automattic/jetpack/pull/10742/files#diff-58180905242554a749dbb58ba9464cc9R435)?

Reply:
Yes since we want to make sure that we don't break anything. 
I created an issue to remove it at some point completely in the future. 
https://github.com/Automattic/jetpack/issues/10748

Reply:
Think there's a big risk in removing them? Looks like they're really only used as callbacks for the endpoint. I'm kinda wary that leaving the dupes around might cause them to proliferate the next time someone touches these files (I'd be kinda confident that removing them shouldn't pose a very big risk).

Reply:
Definitely not a big risk, I'd be surprised if anyone but us were using them. But we're a bit more comfortable leaving them in through the next release just in case - especially since we declare support for versions `current Jetpack -1`.

Reply:
Okay! I'd kinda overlooked the `_deprecated_function()` calls. Works for me!